patches/third_party-blink-renderer-core-dom-document.cc.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
-index 02e6831f00c14a9aab6ba6d4218a94ec133c6076..091eb1776f2ae37992024de1e641ed64b3becbe9 100644
+index 2247f71c3c97..7eaca00dc6f2 100644
```

Comment:
You should be able to revert the changes to the headers and line numbers here safely as well (everything up to line 228)

Reply:
done